app/javascript/components/request-workflow-status/request-workflow-status-item.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState, useRef } from 'react';
+import {
+  Tag, Loading, Link,
+} from 'carbon-components-react';
+import PropTypes from 'prop-types';
+import { workflowStatusData, workflowStateTypes } from './data';
+import MiqDataTable from '../miq-data-table';
+import NotificationMessage from '../notification-message';
+
+/** Component to render the Workflow status in /miq_request/show/#{id} page */
+const RequestWorkflowStatusItem = ({ recordId }) => {
+  const RELOAD = 2000; // Time interval to reload the RequestWorkflowStatus component.
+  const reloadLimit = 5; // This is to handle the Auto refresh issue causing the the server to burn out with multiple requests.
+  const reloadCount = useRef(0);
+
+  const [data, setData] = useState(
+    {
+      isLoading: true,
+      responseData: undefined,
+      message: undefined,
+      list: [],
+      parentName: undefined,
+      validType: false,
+    }
+  );
+
+  /** Function to get the Workflow */
+  const getWorkflow = async() => {
+    reloadCount.current += 1;
+    const url = `/api/configuration_scripts/${recordId}`;
```

Comment:
Is the assumption here that all requests tasks will have the same configuration script id?  That's probably not a real case.